dark-debris/src/pages/index.astro

```diff
@@ -22,11 +22,11 @@ import "../CSS/index.css";
 					<ul>
 						<li>
 							<b>File Upload: </b>Click the Choose File button and select the
-							local file you would like to evaluate
+							local file or drag the file onto the file upload field to evaluate.
```

Comment:
Do you we need to say local file? What if a user is storing it on a remote service like OneDrive? How about this text:

Upload File: Drag and drop a file from your device into the file area or select the Browse... button to choose a file.

It seems like we could describe the file area more descriptively but this seems to be more user-centric than what we have right now.

Reply:
Yeah, I agree. I'll make that change the text.